tests/test_env.py

```diff
@@ -229,9 +229,9 @@ def test_process_env_install_from_index_conflicting_reqs(
     assert package_name not in env.process_env.get_installed_packages()
 
 
-@pytest.mark.parametrize("editable", [True, False])
+@pytest.mark.parametrize_any("editable", [True, False])
```

Comment:
This one tests fundamentally different behavior in my opinion. I would leave this as just `parametrize`. (Only this parameter, the other increases the test coverage only by a little.)

Reply:
```suggestion
@pytest.mark.parametrize("editable", [True, False])
```